index.html

```diff
@@ -1,23 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Superhero Memory Game</title>
-    <!-- LINK THE STYLES HERE -->
-  </head>
-  <body>
-    <div>
-      <h1>Superhero Memory Game</h1>
-    </div>
-    <div id="score">
-      <h2>Score</h2>
-      <p>Pairs clicked: <span id="pairs-clicked">0</span></p>
-      <p>Pairs guessed: <span id="pairs-guessed">0</span></p>
-    </div>
-    <div id="memory-board"></div>
 
-    <!-- LINK THE JAVASCRIPT FILES HERE (keep in mind that the order in which you link them MATTERS) -->
-  </body>
-</html>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Superhero Memory Game</title>
+  <!-- LINK THE STYLES HERE -->
+  <link rel="stylesheet" href="styles/style.css" />
+
+
+</head>
+
+<body>
+  <div>
+    <h1>Superhero Memory Game</h1>
+  </div>
+  <div id="score">
+    <h2>Score</h2>
+    <p>Pairs clicked: <span id="pairs-clicked">0</span></p>
+    <p>Pairs guessed: <span id="pairs-guessed">0</span></p>
+  </div>
+  <div id="memory-board"></div>
+
+  <!-- LINK THE JAVASCRIPT FILES HERE (keep in mind that the order in which you link them MATTERS) -->
+  <script src="./js/memory.js"></script>
+  <script src="./js/index.js"></script>
+</body>
+
+</html>
```

Comment:
Good stuff Sophie!
The UI is not working entirely as expected yet. Some cards remain visible although no pair was clicked. Do you experience the same behaviour? Can it be improved?
Please also check out the comments.